### DISCUSSION TITLE
options for Graph vertex_config

### DISCUSSION THREAD

**Discussion by billsmithatg** [ManimCommunity/manim#2298] [Q&A]
What options are available for a Graph's `vertex_config`?  I see a few examples but no documentation for the full set of options.  Thanks!

**Comment by behackl** [comment]
`vertex_config` is a dictionary holding keyword arguments for the mobject specified via the `vertex_type` keyword argument; by default, for unlabeled graphs this is a [`Dot`](https://docs.manim.community/en/stable/reference/manim.mobject.geometry.Dot.html), and for labeled graphs it is a [`LabeledDot`](https://docs.manim.community/en/stable/reference/manim.mobject.geometry.LabeledDot.html).

Have a look at these classes (and their corresponding base classes) to see all the available arguments.
